map/apply-scheduled-updates-to-preplanned-map-area/gradlew

```diff
@@ -7,7 +7,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      https://www.apache.org/licenses/LICENSE-2.0
+#      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
just curious about this change: you mentioned that this PR doesn't address the Apache licenses, so is this to bring this sample in line with the way the other license references are written?

Reply:
This wasn't intentional. I've done a find&replace to make these `https` originally, and then F&R to put them all back to `http`. So looks like this one was https in the first place. We can change it or leave it, it doesn't really matter I think.